src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/casenotes/CaseNotesSynchronisationService.kt

```diff
@@ -263,8 +259,24 @@ Also add new mappings for the new booking id for the copied case notes, which po
           )
         }
 
+      // Now exclude any mappings that already exist.  This can happen if two merges occur within half an hour of each
+      // other as we find all merged case notes within that time frame
+      val newUniqueMappings = newMappings.filter { it.nomisCaseNoteId !in existingMappings.keys }
+      if (newUniqueMappings.size != newMappings.size) {
+        telemetryClient.trackEvent(
+          "casenotes-prisoner-merge-existing-mappings",
+          mapOf(
+            "offenderNo" to nomsNumber,
+            "removedOffenderNo" to removedNomsNumber,
+            "bookingId" to bookingId,
+            "mappingsCount" to (newMappings.size - newUniqueMappings.size),
```

Comment:
log both the count of mappings we have ignored, as well as the case note ids of them